# 补血点

# 1号补血点是否被占领
bool isFirstOccupied
# 2号补血点是否被占领
bool isSecondOccupied
# 3号补血点是否被占领
bool isThirdOccupied